usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -2412,6 +2414,139 @@ CK_RV token_specific_rng(STDLL_TokData_t * tokdata, CK_BYTE * output,
     return rc;
 }
 
+static CK_BBOOL ep11tok_is_blob_new_wkid(STDLL_TokData_t *tokdata,
+                                         CK_BYTE *blob, CK_ULONG blob_len)
```

Comment:
blob could be const here, but unfortunately it's also not const in ber_decode_SEQUENCE ...

Reply:
right. If I make it const, I would need to cast it no non-const for the ber_decode_SEQUENCE  call......
Will leave it as is for now.
I do have a general 'constification' rework on my TODO list, but this will happen later.